sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipRoutingClient.cs

```diff
@@ -265,7 +331,53 @@ public virtual Response<IReadOnlyList<SipTrunk>> GetTrunks(
             scope.Start();
             try
             {
-                var response = _restClient.GetSipConfiguration(cancellationToken);
+                var response = _restClient.Get(cancellationToken: cancellationToken);
+                return Response.FromValue((IReadOnlyList<SipTrunk>)response.Value.Trunks.Values.ToList().AsReadOnly(), response.GetRawResponse());
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Get List of configured <see cref="SipTrunk"/>.
+        /// </summary>
+        /// <param name="expand">Expand enum.</param>
+        /// <param name="cancellationToken">Optional cancellation token.</param>
+        /// <returns>List of configured trunks.</returns>
+        public virtual Response<IReadOnlyList<SipTrunk>> GetTrunks(
+            ExpandEnum expand,
```

Comment:
Feedback from review:
don't use ExpandEnum in public methods, let's use:
 boolean includeHealth parameter instead.

Reply:
Also, remove the CancellationToken parameter.